### PR TITLE
Add Ubuntu 22.04 support

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: spec/acceptance/puppetserver_upgrade_spec.rb [hunk 1]
```diff
@@ -16,7 +16,12 @@
 
     case ENV['BEAKER_PUPPET_COLLECTION']
     when 'puppet7'
-      from_version = '7.6.0'
+      from_version =
+        if fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04'
+          '7.13.0'
```
Comment (evgeni) [inline spec/acceptance/puppetserver_upgrade_spec.rb:21]: this makes the upgrade test on Ubuntu 22 essentially a noop, but at least it passes now :D

File: spec/acceptance/puppetserver_upgrade_spec.rb [hunk 2]
```diff
@@ -17,7 +17,8 @@
     case ENV['BEAKER_PUPPET_COLLECTION']
     when 'puppet7'
       from_version =
-        if fact('os.family') == 'RedHat' && fact('os.release.major') == '9'
+        if (fact('os.family') == 'RedHat' && fact('os.release.major') == '9') ||
+          (fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04')
           '7.13.0'
```
Comment (ekohl) [inline spec/acceptance/puppetserver_upgrade_spec.rb:22]: Like https://github.com/theforeman/puppet-puppet/pull/893#discussion_r1382209344. Perhaps we should skip the whole test if there is just a single version?

Comment (evgeni) [reply spec/acceptance/puppetserver_upgrade_spec.rb:22]: I was hoping 7.14 just pops up soon and we can use 13->14 as the test.
But yeah, in the meantime we can also skip it totally for EL9 and U22.

File: spec/acceptance/puppetserver_upgrade_spec.rb [hunk 3]
```diff
@@ -1,6 +1,8 @@
 require 'spec_helper_acceptance'
 
-unless unsupported_puppetserver
+unless unsupported_puppetserver ||
+    (fact('os.family') == 'RedHat' && fact('os.release.major') == '9') ||
+    (fact('os.name') == 'Ubuntu' && fact('os.release.major') == '22.04')
```
Comment (evgeni) [inline spec/acceptance/puppetserver_upgrade_spec.rb:5]: or should I move this to a `unsupported_puppetserver_upgrade` helper?

Comment (ekohl) [reply spec/acceptance/puppetserver_upgrade_spec.rb:5]: I was also thinking about that.

**Review by ekohl — APPROVED** [review]

**Pull request merged**